coralproject/cay
# Fixes form builder arrows. See #394

## What does this PR do?
Fixes the form builder arrows.

## How do I test this PR?
- Go to **Create form**
- Add some fields
- Move 'em around using the arrows -> They should move.
